setup.py

```diff
@@ -12,4 +12,22 @@
     test_suite="tests",
     use_2to3=True,
     requires=['slimit'],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    classifiers=[
```

Comment:
Last time I pushed a classifier change (to troposphere) it broke PyPI and I applied this [change](https://github.com/cloudtools/troposphere/commit/47138aa1928aa759a032271801eb105f7c9d7c5e). I believe your change matches the fixed version.

Reply:
Yeah, I think I copied it from troposphere.